src/streamlink_cli/output.py

```diff
@@ -99,6 +103,62 @@ def running(self):
         sleep(0.5)
         return self.player.poll() is None
 
+    @classmethod
+    def supported_player(cls, cmd):
+        """
+        Check if the current player supports adding a title
+
+        :param cmd: command to test
+        :return: name of the player|None
+        """
+        if not is_win32:
+            # under a POSIX system use shlex to find the actual command
+            # under windows this is not an issue because executables end in .exe
+            cmd = shlex.split(cmd)[0]
+
+        cmd = os.path.basename(cmd.lower())
+        for player, possiblecmds in SUPPORTED_PLAYERS.items():
+            for possiblecmd in possiblecmds:
+                if cmd.startswith(possiblecmd):
+                    return player
+
+    @classmethod
+    def _mpv_title_escape(cls, title_string):
+        # mpv has a "disable property-expansion" token which must be handled in order to accurately represent $$ in title
+        if '\$>' in title_string:
+            processed_title = ""
+            double_dollars = True
+            i = dollars = 0
+            while i < len(title_string):
+                if double_dollars:
+                    if title_string[i] == "\\":
+                        if title_string[i+1] == "$":
+                            processed_title += "$"
+                            dollars += 1
+                            i += 1
+                            if title_string[i+1] == ">" and dollars % 2 == 1:
+                                double_dollars = False
+                                processed_title += ">"
+                                i += 1
+                        else:
+                            processed_title += "\\"
+                    elif title_string[i] == "$":
+                        processed_title += "$$"
+                    else:
+                        dollars = 0
+                        processed_title += title_string[i]
+                else:
+                    if title_string[i:i+2] == "\\$":
```

Comment:
`E226 missing whitespace around arithmetic operator`

could you use `[i:i + 2]`